packages/froala-editor/package.json

```diff
@@ -40,22 +40,23 @@
     "@latticejs/storybook-readme": "^1.0.1-beta.1",
     "@material-ui/core": "^4.8.2",
     "@material-ui/icons": "^4.5.1",
-    "@storybook/addon-actions": "^5.3.19",
-    "@storybook/addon-storyshots": "^5.3.19",
-    "@storybook/addon-storysource": "^5.3.19",
-    "@storybook/react": "^5.3.19",
+    "@storybook/addon-actions": "^6.0.28",
+    "@storybook/addon-storyshots": "^6.0.28",
+    "@storybook/addon-storysource": "^6.0.28",
+    "@storybook/react": "^6.0.28",
     "enzyme": "^3.10.0",
     "enzyme-adapter-react-16": "^1.15.1",
     "jest": "^23.6.0",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
     "react-test-renderer": "^16.13.1",
     "rimraf": "^2.7.0",
-    "storybook-readme": "^4.0.5",
+    "storybook-readme": "^5.0.8",
     "typeface-montserrat": "0.0.75",
     "typeface-roboto": "0.0.75"
   },
   "dependencies": {
+    "@storybook/addon-essentials": "^6.0.28",
```

Comment:
All StoryBook dependencies should come inside dev dependencies.

Reply:
All things fixed